packages/gitmoji-changelog-markdown/src/index.js

```diff
@@ -57,23 +58,26 @@ function markdownIncremental({ meta, changes }, options) {
             null,
             string.split('\n')
               .reduce(
-                (content, nextLine) => {
-                  previousNextFound = previousNextFound || nextLine.startsWith(`<a name="${release}"></a>`)
-                  previousVersionFound = nextLine.startsWith(`<a name="${lastVersion}"></a>`)
+                (content, nextLine, index, array) => {
+                  previousVersionFound = matchVersionBreakpoint(nextLine, lastVersion)
```

Comment:
What about?
```
if (nextVersionWritten) {
// [line 77 to 80]
}

// [line 62 to 74]
```

Reply:
No I can't because of other conditions of lines 62 to 74.
Else I should do, and there will be duplication of code:
```
if (nextVersionWritten) {
// [line 77 to 80]
}

// [line 62 to 74]

// [line 77 to 80]
```

Reply:
- Line 66, you pass the condition only if nextVersionWritten is FALSE (&& usage)
 - Line 71 you pass the condition only if  nextVersionWritten is FALSE (&& usage)

This is more:
If it doesn't work, juste resolve the conversation ;)
```js
[line 62 to 63]

if (nextVersionWritten) {
// [line 77 to 80]
}

[line 66 to 74]
```